infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/LogReplicationConfig.java

```diff
@@ -39,7 +39,7 @@ public class LogReplicationConfig {
 
     // Max message size supported by protocol buffers is 64MB. Log Replication uses this limit as the default message
     // size to batch and send data across over to the other side.
-    public static final int MAX_DATA_MSG_SIZE_SUPPORTED = (64 << 20);
+    public static final int MAX_DATA_MSG_SIZE_SUPPORTED = (24 << 20);
```

Comment:
Shouldnt this value be 25 MB?  Can you also update the comment to reflect the change and how 25 MB was decided (default max tx size supported for Corfu tx)